test.mk

```diff
@@ -236,6 +240,30 @@ uefi-aa64-test:
 	rm -rf test_image loopback_dev
 	qemu-system-aarch64 -m 512M -M virt -cpu cortex-a72 -bios ovmf-aa64/OVMF.fd -net none -smp 4 -device ramfb -device qemu-xhci -device usb-kbd  -hda test.hdd -serial stdio
 
+.PHONY: uefi-rv64-test
+uefi-rv64-test:
+	$(MAKE) ovmf-rv64
+	$(MAKE) test-clean
+	$(MAKE) test.hdd
+	$(MAKE) limine-uefi-riscv64
+	$(MAKE) -C test TOOLCHAIN_FILE='$(call SHESCAPE,$(BUILDDIR))/toolchain-files/uefi-riscv64-toolchain.mk'
```

Comment:
These `test` targets are apparently broken outside of x86-64, but I commented this out to test the rest of the rule with no payload kernels.

Reply:
This *should* now work with the latest commit.